blazor/diagram/migration-classic-to-native.md

```diff
@@ -0,0 +1,577 @@
+| Sets the Template property for the annotation of the connector | **Property:** *Connectors[i].Annotations[i].Template* <br><br>`<SfDiagram Connectors="ConnectorCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public ConnectorCollection = new ObservableCollection<DiagramConnector>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`DiagramConnector connector1 = new DiagramConnector()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`SourcePoint=new ConnectorSourcePoint(){X = 100, Y = 100},`<br>&nbsp;&nbsp;&nbsp;`TargetPoint = new ConnectorTargetPoint(){X=200, Y=200},`<br>&nbsp;&nbsp;&nbsp;`Annotations = new ObservableCollection<DiagramConnectorAnnotation>(){ new DiagramConnectorAnnotation(){ Content = "label", AnnotationType = AnnotationType.Template, Template = "<div>Annotation template</div>" } }`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`ConnectorCollection.Add(connector1);`<br>&nbsp;`}`<br>`}`| Not supported |
+| Sets the Type property for the node | **Property:** *Connectors[i].Annotations[i].Type* <br><br>`<SfDiagram Connectors="ConnectorCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public ConnectorCollection = new ObservableCollection<DiagramConnector>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`DiagramConnector connector1 = new DiagramConnector()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`SourcePoint=new ConnectorSourcePoint(){X = 100, Y = 100},`<br>&nbsp;&nbsp;&nbsp;`TargetPoint = new ConnectorTargetPoint(){X=200, Y=200},`<br>&nbsp;&nbsp;&nbsp;`Annotations = new ObservableCollection<DiagramConnectorAnnotation>(){ new DiagramConnectorAnnotation(){ Content = "label", Type = AnnotationTypes.Path } }`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`ConnectorCollection.Add(connector1);`<br>&nbsp;`}`<br>`}`| Not supported |
+
+#### Fixed User handle for Connector
+| Behavior | API in SfDiagram | API in SfDiagramComponent |
+| -------- | -------- | -------- |
+| Defines a collection of fixed user handle to be render in the connector | **Property:** *Connectors[i].FixedUserHandles* <br><br>Sets a FixedUserHandles property as *ObservableCollection*<*DiagramConnectorFixedUserHandle*> <br><br>`<SfDiagram Connectors="ConnectorCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public ConnectorCollection = new ObservableCollection<DiagramConnector>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`DiagramConnector connector1 = new DiagramConnector()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbs
```

Comment:
| Triggers when a symbol is dragged and dropped from the symbol palette to the drawing area | **Event** *OnDrop* <br>`<SfDiagram @ref="@Diagram" Height="600px">`<br>&nbsp;`<DiagramEvents OnDrop ="@OnDrop">`<br>`</SfDiagram>`<br>`@code`<br>`{`<br>&nbsp;`SfDiagram Diagram;`<br>&nbsp;`private void OnDrop(IBlazorDropEventArgs  args)`<br>&nbsp;`{`<br>`}`<br>`}` | **Event** *DragDrop* <br>`<SfDiagramComponent DragDrop="OnDragDrop"/>`<br>`@code`<br>`{`<br>&nbsp;`SfDiagram Diagram;`<br>&nbsp;`private void OnDragDrop(DropEventArgs args)`<br>&nbsp;`{}`<br>`}` |